.circleci/config.yml

```diff
@@ -14,175 +14,94 @@ aliases:
           name: Install Node modules
           command: npm i -ci
-
       - save_cache:
           key: v1-dependencies-{{ checksum "package-lock.json" }}
           paths:
             - node_modules
 
   build-pwa:
-    docker:
-      - image: circleci/node:10
-
-    working_directory: ~/repo
-
+    <<: *defaults
     steps:
       - checkout
       - restore_cache: *restore-npm-cache
       - restore_cache: *restore-webpack-cache
-
       - run:
           name: Build Project
           command: npm run build
-
       - save_cache:
           key: v1-dist-{{ .Environment.CIRCLE_SHA1 }}
           paths:
             - build
-
       - save_cache:
           key: v1-webpackcache-{{ checksum "package.json" }}
           paths:
             - node_modules/.cache
 
-  check-sizes:
-    docker:
-      - image: circleci/node:10
-
-    working_directory: ~/repo
-
+  lint:
+    <<: *defaults
     steps:
       - checkout
-      - restore_cache: *restore-dist-cache
-      - restore_cache: *restore-bundlesize
-
+      - restore_cache: *restore-npm-cache
       - run:
-          name: Check Sizes
-          command: npm run bundlesize
-
-      - save_cache:
-          key: v1-bundlesize-master
-          paths:
-            - node_modules
+          name: Run Unprefixed Version Check
+          command: npm run check:versions
+      - run:
+          name: Run TSLint
+          command: npm run lint
+      - run:
+          name: Run Test Typescript Compile
+          command: npm run check:compile
 
-  lighthouse:
+  gimbal:
+    <<: *defaults
     docker:
-      - image: circleci/node:10-browsers
-
-    working_directory: ~/repo
-
+      - image: circleci/node:latest-browsers
     steps:
       - checkout
       - restore_cache: *restore-dist-cache
-      - restore_cache: *restore-lighthouse
-
-      - run:
-          name: Setup Artifacts
-          command: mkdir ./artifacts
-
-      - run:
-          name: Install Lighthouse
-          command: npm i lighthouse serve circle-github-bot
-
-      - run:
-          name: Serve the app
-          command: yarn serve -l 3000 -s build
-          background: true
-
-      # wait for the server to start responding.
       - run:
-          shell: /bin/sh
+          name: Setup npm
           command: |
-            wget --retry-connrefused --waitretry=1 --read-timeout=20 --timeout=15 -t 10 http://localhost:3000
-            :
-
+            printf "@modus:registry=https://npm.modus.app\n//npm.modus.app/:_authToken=\"%s\"" "$MODUS_NPM_TOKEN" >> ~/.npmrc
+            npm set prefix=/home/circleci/npm
+            echo 'export PATH=/home/circleci/npm/bin:$PATH' >> $BASH_ENV
+            source $BASH_ENV
       - run:
-          name: Run Lighthouse
-          command: yarn lighthouse "http://localhost:3000/?moduslabsci"  --output json --output html --output-path=./artifacts/lighthouse.json --config-path=./lighthouse.config.js
-
+          name: Install Gimbal
+          command: npm install -g @modus/gimbal
```

Comment:
I think it'd be easier to just `npm i -g @modus/gimbal --registry https://npm.modus.app` and avoid using `.npmrc`